src/GeneratorHelper.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+/*
+ * This file is part of the Symfony MakerBundle package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MakerBundle;
+
+/**
+ * @author Sadicov Vladimir <sadikoff@gmail.com>
+ */
+class GeneratorHelper
+{
+    public function getFormFieldsPrintCode(array $formFields): string
+    {
+        $printCode = '';
+
+        foreach ($formFields as $fieldName) {
+            $printCode .= <<<PHP
+            ->add('$fieldName')
+PHP;
+        }
```

Comment:
I'd actually prefer for this to live inside the template: it's not very complex - just a loop.

Reply:
agreed, don't know why I added this loop in helper :)